nomad-gcp/modules/nomad-server-gcp/firewall.tf

```diff
@@ -1,6 +1,6 @@
 
 resource "google_compute_firewall" "nomad" {
-  name    = "fw-${var.name}-allow-nomad-client-traffic-circleci-server"
+  name    = "${var.name}-circleci-allow-nomad-client-traffic-nomad-servers"
```

Comment:
Shorten the firewall name to abide with max 62 chars allowed for firewall rule name in GCP